src/codeu/chat/client/core/View.java

```diff
@@ -136,4 +137,22 @@ public View(ConnectionSource source) {
 
     return messages;
   }
+
+  public ServerInfo getInfo() {
+    try (final Connection connection = this.source.connect()) {
+      Serializers.INTEGER.write(connection.out(), NetworkCode.SERVER_INFO_REQUEST);
+      if (Serializers.INTEGER.read(connection.in()) == NetworkCode.SERVER_INFO_RESPONSE) {
+        final Uuid version = Uuid.SERIALIZER.read(connection.in());
+        final Time startTime = Time.SERIALIZER.read(connection.in());
```

Comment:
Nit:  we have two reads in one class that depend on the order they are written in another. Consider how you could reduce the coupling between these classes, and/or alert future devs that if this changes, the other should as well.